engines/bops_api/app/views/bops_api/v2/public/_show.json.jbuilder

```diff
@@ -12,6 +12,8 @@ json.application do
   json.fullReference planning_application.reference_in_full
   json.received_at planning_application.received_at
   json.receivedAt planning_application.received_at
+  json.valid_from planning_application.validated_at
+  json.published_at planning_application.published_at
```

Comment:
Can we use camel case here, I'm enforcing it with my submission endpoint PR

Reply:
Sure thing — I remembered we were trying to move from one to the other but lost track of which way around we were going 🤦🏻